service/test/unit/test_account_recovery.py

```diff
@@ -53,7 +75,7 @@ def test_update_recovery_code(self):
     @defer.inlineCallbacks
     def test_send_recovery_code_by_email(self):
         sender = 'team@{}'.format(self.domain)
-        msg = MIMEText('Your code %s' % self.generated_code)
+        msg = MIMEText(RECOVERY_CODE_EMAIL)
```

Comment:
Instead of testing the functionality of retrieving the document, can we mock `pkg_resources.resource_filename()` so we don't have to reproduce the whole email above? That way, if the text changes, our test won't fail. We can test just that the variables were inserted correctly.

Or, maybe we don't need to mock, but just change the assert to only assert that the variables exist (let `mock_sendmail.assert_called_with()` with `ANY` as the last variable, and a second assert, for example).